packages/cli/generators/controller/index.js

```diff
@@ -236,13 +236,7 @@ module.exports = class ControllerGenerator extends ArtifactGenerator {
       debug(`artifactInfo: ${inspect(this.artifactInfo)}`);
       debug(`Copying artifact to: ${dest}`);
     }
-    this.fs.copyTpl(
-      source,
-      dest,
-      this.artifactInfo,
-      {},
-      {globOptions: {dot: true}},
-    );
+    this.copyTpl(source, dest, this.artifactInfo);
```

Comment:
Can we avoid abbreviations in our names please? I am proposing `copyTemplate` or `copyTemplates` depending on whether this method copies a single or multiple templates.

Reply:
Renamed to `copyTemplatedFiles`